<ui version="4.0" >
 <class>BadPixelBase</class>
 <widget class="QDialog" name="BadPixelBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Bad Pixel List</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="Headline" >
       <property name="text" >
        <string>Bad pixel list:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Q3Table" name="PixelList" >
       <property name="numRows" >
        <number>3</number>
       </property>
       <property name="numCols" >
        <number>3</number>
       </property>
       <row>
        <property name="text" >
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text" >
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text" >
         <string>3</string>
        </property>
       </row>
       <column>
        <property name="text" >
         <string>1</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>2</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>3</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="WrtieButt" >
         <property name="text" >
          <string>Write to file</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer1" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="MaskButt" >
         <property name="text" >
          <string>Create mask</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Spacer3" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CloseButt" >
         <property name="text" >
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
